### PR TITLE
update priv network v3

### LINKED ISSUE #885: Review & update Tutorial: Start a Private Network with Substrate to v3 compatible
https://substrate.dev/docs/en/tutorials/start-a-private-network/

### PR DESCRIPTION
**Pull request by NukeManDan**
All is updated but I can't get finality to work as described. I will update when this works, but it is ready for a review on everything but updating the example output of the node (and any new instructions if needed) 

Closes #885

### CODE REVIEW

**Comment by NukeManDan** [comment]
@sacha-l were you able to get finality working?

**Comment by riusricardo** [comment]
I need to take a look to this but it looks good in general. @NukeManDan after adding the grandpa keys, did you stopped and restarted the node?

**Comment by NukeManDan** [comment]
I did, my issue was incorrectly inserting the wrong key I think. 

I have added a bit more clarifications to help, and a solution referenced with known good chain spec files and helpers to get keys inserted if anyone gets stuck like me 😅 
https://github.com/substrate-developer-hub/substrate-node-template/tree/tutorials/solutions%2Fprivate-chain-v3

File: docs/tutorials/start-a-private-network/customchain.md [hunk 1]
```diff
@@ -145,93 +155,132 @@ If you enter the command and parameters correctly, the node will return a JSON r
 
 Make sure you delete the file that contains the keys when you are done.
 
+<!-- TODO: add option 3: use the embedded `key` CLI tool -->
+
+### Verify Keys in the Keystore
```
Comment (NukeManDan) [inline docs/tutorials/start-a-private-network/customchain.md:160]: @riusricardo this is a new section to review, with a TODO in comments ( a few more predate me working on this) to add more details latter

Comment (riusricardo) [inline docs/tutorials/start-a-private-network/customchain.md:160]:
```suggestion
### Verify Keys in the Keystore (Optional)
```

**Review by riusricardo — APPROVED** [review]
Nice! Very good enhancements :+1:

**Pull request merged**